### DISCUSSION TITLE
Use of rules to match the ignition of multiple relays

### DISCUSSION THREAD

**Discussion by MaxRich** [arendst/Tasmota#16258] [Q&A] [answered]
Hello everybody. I have recently been experimenting with Tasmota on ESP32 to solve some home automation problems with curtains that must be controlled through radio commands with clean contacts.
For the ascent and descent command I have configured Relay1 and Relay2. The problem arises with the stop command for which I have to activate both Relay2 and Relay3 at the same time.
I was thinking of configuring another door by associating it with the stop command and activating rules to manage it. Unfortunately, not being very familiar with these commands, I can't get it to work.
Is there anyone who can help me?

I thought I was doing something like this:
rule1 ON Power3 # state = 1 DO Power1 = 1 ENDON ON Power3 # state = 1 DO Power2 = 1 ENDON
rule1 ON Power3 # state = 0 DO Power1 = 0 ENDON ON Power3 # state = 1 DO Power2 = 0 ENDON

**Comment by sfromis** [comment, marked as answer]
Your rules have bad syntax, the Power command takes no equals sign. And better not have extra spaces around # and =. If you entered those two as commands, the 2nd `rule1` command will overwrite the first, and you may not have activated the rule set. Shorter:
`rule1 ON Power3#state=1 DO Backlog Power1 1; Power2 1 ENDON`

**Reply by MaxRich** [reply]
Thank you very much! I would say it does what I was asking but response times are very slow. In fact, by setting a "PulseTime" of 1 second, it cannot make the relays switch on at the same time and therefore cannot send the stop command. Do you have any idea how I could solve it without having to dedicate a third relay that drives both of them?

**Reply by sfromis** [reply]
Well, if it is the rules above, I just do not understand what you are trying to do. You have 3 triggers `Power3#state=1` but only one  `Power3#state=0`.

**Reply by pkkrusty** [reply]
There is a default delay of 200ms in between backlog commands, which you can reduce to make things happen more quickly. I forget which setoption it is.

**Reply by barbudor** [reply]
`Backlog0` have no delay between commands

**Reply by MaxRich** [reply]
Thank you very much.  The result is very close to what I wanted to achieve, too bad the delay command with baclock0 doesn't seem to work.  I would have liked to keep at least 1 second both channels active.  Anyway I would say that it is already a great result for me.
